source/braille.py

```diff
@@ -959,6 +979,7 @@ def _addTextWithFields(self, info, formatConfig, isSelection=False):
 			# We only render fields that aren't at the start of their nodes for the first part of the reading unit.
 			# Otherwise, we'll render fields that have already been rendered.
 			self._skipFieldsNotAtStartOfNode = True
+		info.obj._brailleFormatFieldAttributesCache = formatFieldAttributesCache

```

Comment:
> My concern is that this will clutter many NVDAObjects with empty format field caches.

Only objects which were at some point the focus or navigator object, and no more clutter than `speakTextInfo` produces. I guess you could test whether the updated cache is empty and if so, ensure the NVDAObject attribute is removed. However, I don't think there's really any point.